### PR TITLE
Add AddPermissionsForUser api for syncedEnforcer

### PR DESCRIPTION
**Pull request by jaxsong**

### CODE REVIEW

File: rbac_api_synced.go [hunk 1]
```diff
@@ -99,6 +99,14 @@ func (e *SyncedEnforcer) AddPermissionForUser(user string, permission ...string)
 	return e.Enforcer.AddPermissionForUser(user, permission...)
 }
 
+// AddPermissionsForUser adds multiple permissions for a user or role.
+// Returns false if the user or role already has one of the permissions (aka not affected).
+func (e *SyncedEnforcer) AddPermissionsForUser(user string, permission ...[]string) (bool, error) {
```
Comment (yanlong-turingvideo) [inline rbac_api_synced.go:104]: 添加已存在的permission返回false, err=nil?

Comment (jaxsong) [reply rbac_api_synced.go:104]: 是的

Comment (longshine) [inline rbac_api_synced.go:104]: What's the difference compared to the other `AddPermissionForUser(user string, permission ...string)`?

**Review by longshine — APPROVED** [review]
LGTM

**Review by yanlong-turingvideo — APPROVED** [review]

**Pull request merged**